omnibus/config/software/supermarket-cookbooks.rb

```diff
@@ -41,6 +41,7 @@
     open("#{cookbooks_path}/solo.rb", "w") do |file|
       file.write <<-EOH.gsub(/^ {8}/, '')
         cookbook_path   "#{cookbooks_path}"
+        cache_path "/var/opt/supermarket/local-mode-cache"
```

Comment:
@schisamo By lock step, do you mean that both `cache_path` and `file_cache_path` should point to the same directory under `/var/opt/supermarket`? Or should it be something like:

```ruby
file_cache_path "/var/opt/supermarket/cache"
cache_path      "/var/opt/supermarket/cache/local-mode"
```

@sersut All these paths I'm just finding out about!